grammars/repositories/flavors/rmarkdown.cson

```diff
@@ -5,27 +5,20 @@
 key: 'rmarkdown'
 patterns: [
   {
-    begin: '(?x)
-      (```)(
-        (\\{)(r)(?:\\s)?
-        (?:([a-zA-Z0-9|_|-]*)(?=[\\s|,|}])\\s?)?
-        ([^\\}]*)?
-        (\\})
-      )
-    '
-    end: '^(\\1)$'
+    begin: '(?:\\n|^)(`{3,})[ \\.]*(\\{)(\\S*) *(?:([^= ]*) *(?=[,\\}]))? *,?(.*?)(\\}) *(?=\\n|$)'
```

Comment:
Could you rewrite this bit so it's separated over multiple lines as it was in the previous version? Makes it a lot easier to read regular expressions, and easier to review for me what you've changed.